lib/node/http.js

```diff
@@ -255,6 +255,23 @@ class HTTP extends Server {
       res.json(200, block.getJSON(this.network, view, height, depth));
     });
 
+    // Block Header by hash/height
+    this.get('/header/:block', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const hash = valid.uintbrhash('block');
+
+      enforce(hash != null, 'Hash or height required.');
+
+      const entry = await this.chain.getEntry(hash);
+
+      if (!entry) {
+        res.json(404);
+        return;
+      }
+
+      res.json(200, entry.toJSON());
```

Comment:
Any reason this is the `ChainEntry` instead of `Headers`? This will also include `chainwork` in the result, otherwise should currently be the same.

Reply:
FWIW, as @pinheadmz mentioned, this is [available as an RPC method](http://bcoin.io/api-docs/index.html#getblockbyheight) and in that version it also returns the chain work.

Reply:
Ah, you mean `getblockheader`? Looks like `getblockbyheight`, `getblock` and `getblockheader` all return `chainwork`.